sdk/python/feast/cli.py

```diff
@@ -68,6 +73,15 @@ def cli(ctx: click.Context, chdir: str):
     """
     ctx.ensure_object(dict)
     ctx.obj["CHDIR"] = Path.cwd() if chdir is None else Path(chdir).absolute()
+    try:
+        level = getattr(logging, log_level.upper())
+        logging.basicConfig(
```

Comment:
does this override the info levels you set for other files?

also mind changing the PR description to be more specific to logging refactors for serverless?

Reply:
I removed the per-file configuration. Now all loggers will conform to the level set here. And yes, changed PR description.